components/accordion/index.css

```diff
@@ -84,9 +84,7 @@
 	}
 
 	/* CJK (Chinese, Japanese, and Korean) language support */
-	&:lang(ja),
-	&:lang(zh),
-	&:lang(ko) {
+	&:lang(ja, zh, ko) {
```

Comment:
I wasn't seeing this working in my browser, it seems like that might be because it's not supported in Chrome yet 😿 

Should we hold on making this update for now?

Reply:
Oh dang! Yes, great call.